README.md

```diff
@@ -54,248 +99,292 @@ $(SRCROOT)/Carthage/Build/iOS/CryptoSwift.framework
 
-The `setupDevice` API should be called after user login or signup is successful.
 
-Once the user is logged in, then `setupDevice` should be called every time the app launches, this ensures that the current device is registered before communicating with OST Platform server.<br/><br/>
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_tokenId: Token id provided by application server_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
 
-```Swift
+<br>
+### 2. setupDevice
+This workflow needs `userId` and `tokenId` so `setupDevice` should be called after your app login or signup is successful.
+Using the mapping between userId in OST Platform and your app user, you have access to `userId` and `tokenId`.
+
+**If the user is logged in, then `setupDevice` should be called every time the app launches, this ensures that the current device is registered before communicating with OST Platform server.**
+
+
+```
 OstWalletSdk.setupDevice(
     userId: String,
     tokenId: String,
     delegate: OstWorkflowDelegate
-    )
-```    
+)
+```
+
+| Parameter | Description |
+|---|---|
+| **userId** <br> **String**	| Unique identifier of the user stored in OST Platform  |
+| **tokenId** <br> **String**	| Unique identifier of the token economy |
+| **delegate** <br> **OstWorkflowDelegate**	|An object that implements the callback function available in `OstWorkflowDelegate` protocol. These callback functions are needed for communication between app and wallet SDK. Implement `flowComplete` and `flowInterrupt` callback functions to get the workflow status. Details about other callback function can be found in [OstWorkflowDelegate protocol reference](/platform/docs/sdk/references/wallet_sdk/iOS/latest/protocols/).<br> This object should implement `registerDevice` callback function. `registerDevice` will be called during the execution of this workflow.  |
 
-### Activate the user
+<br>
 
-User activation refers to the deployment of smart-contracts that form the user's Brand Token wallet. An activated user can engage with a Brand Token economy.<br/><br/>
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_pin: User Pin_<br/>
-&nbsp;_passphrasePrefix: Passphrase prefix provided by application server_<br/>
-&nbsp;_spendingLimit: Spending limit in a transaction in atto BT_<br/>
-&nbsp;_expireAfterInSec: Session key validat duration_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
+### 3. activateUser
+It `authorizes` the registered device and activates the user. User activation deploys  **TokenHolder**, Device manager  contracts on blockchain. Session keys are also created and authorized during `activateUser` workflow. So after `user activation`, users can perform wallet actions like executing transactions and reset pin. 
```

Comment:
To me, this is ambiguous.
activateUser needs to be done only once per user irrespective of the number of devices they own. 
By stating: 
It `authorizes` the registered device 
I think developers will miss the point that each device needs to authorized separately.

I think more clear description of the workflow is needed here.